lib/http3/common.c

```diff
@@ -134,8 +179,11 @@ int h2o_http3_send_datagram(h2o_http3_ctx_t *ctx, quicly_datagram_t *p)
 
         /* Temporary failure to send a packet is not a permanent error fo the connection. (TODO do we want do something more
          * specific?) */
-        perror("sendmsg failed");
+        track_sendmsg_failure();
+    } else {
+        track_sendmsg_success();
     }
+    track_sendmsg_flush(ctx->loop);
```

Comment:
I wonder if we need to log anything when there is no loss. To paraphrase, it might make sense to convert `track_sendmsg_flush` into a subroutine of `track_sendmsg_failure`.

The other benefit of merging `track_sendmsg_flush` into `track_sendmsg_failure` is that we can get rid of `last_errno`. `errno` would always be available when we log the message.

Reply:
If we do so, we might wait for a long time for messages to appear, would you be ok with that?

Reply:
How about emitting a message when both of the following conditions are met:
* a new failure event occurs
* at least 60 seconds have elapsed since the last emission of the message

That would guarantee that we'd see the message at least once a minute, when loss is happening. I think that would be fine?